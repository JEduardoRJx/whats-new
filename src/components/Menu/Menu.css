* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

nav {
  display: flex;
  flex-direction: column;
  padding: 20px;
  width: 325px;
}

h1 {
  color: #CCB69B;
  font-size: 2.25em;
}
span {
  color: #AF1B3F;
}

button {
  align-items: center;
  background-color: #473144;
  border-radius: 0px 45px 45px 0px;
  border: none;
  color:#CCB69B;
  display: flex;
  font-size: 2em;
  height: 60px;
  justify-content:space-between;
  margin-top: 40px;
  outline: 0;
  padding: 0px 10px;
}

button:hover,
button:focus {
  background-color: #AF1B3F;
}

.icon {
  border-radius: 0px;
  height: inherit;
  width: auto;
}