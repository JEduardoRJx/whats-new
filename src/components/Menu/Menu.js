import React from 'react';
import './Menu.css'

const Menu = () => {
  return (
    <nav>
      <h1>What's <span>New?</span></h1>
      <button type='button'>IMG Local News</button>
      <button type='button'>IMG Technology</button>
      <button type='button'>IMG Entertainment</button>
      <button type='button'>IMG Science</button>
      <button type='button'>IMG Health</button>
    </nav>
  )
}

export default Menu;