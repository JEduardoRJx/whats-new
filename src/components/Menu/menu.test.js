import React from 'react';
import { shallow } from 'enzyme';
import Menu from './Menu.js'

describe('Menu', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Menu />)
  });

  it('should create a snapshot of menu', () => {
    expect(wrapper).toMatchSnapshot();
  });

});