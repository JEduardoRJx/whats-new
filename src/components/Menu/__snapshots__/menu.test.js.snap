// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should create a snapshot of menu 1`] = `
<nav>
  <h1>
    What's 
    <span>
      New?
    </span>
  </h1>
  <button
    className="localNews menu-btn"
    onClick={[Function]}
    type="button"
  >
    <img
      alt="local icon"
      className="icon"
      src="local.svg"
    />
    Local News
  </button>
  <button
    className="technologyNews menu-btn"
    onClick={[Function]}
    type="button"
  >
    <img
      alt="technology icon"
      className="icon"
      src="technology.svg"
    />
    Technology
  </button>
  <button
    className="entertainmentNews menu-btn"
    onClick={[Function]}
    type="button"
  >
    <img
      alt="entertainment icon"
      className="icon"
      src="entertainment.svg"
    />
    Entertainment
  </button>
  <button
    className="scienceNews menu-btn"
    onClick={[Function]}
    type="button"
  >
    <img
      alt="science icon"
      className="icon"
      src="science.svg"
    />
    Science
  </button>
  <button
    className="healthNews menu-btn"
    onClick={[Function]}
    type="button"
  >
    <img
      alt="health icon"
      className="icon"
      src="health.svg"
    />
    Health
  </button>
</nav>
`;
